static/sass/styles.scss

```diff
@@ -45,161 +45,196 @@ $color-navigation-background: #252525;
 
-@import 'patterns_maas_modal';
+@import "patterns_maas_modal";
 @include maas-p-modal;
 
-@import 'snapcraft_details_heading';
+@import "snapcraft_details_heading";
 @include snapcraft-details-heading;
 
-@import 'snapcraft_footer';
+@import "snapcraft_footer";
 @include snapcraft-footer;
 
-@import 'snapcraft_snap-details';
+@import "snapcraft_snap-details";
 @include snapcraft-snap-details;
 
-@import 'snapcraft_404';
+@import "snapcraft_404";
 @include snapcraft-404;
 
-@import 'snapcraft_p-form-help-text';
+@import "snapcraft_p-form-help-text";
 @include snapcraft-p-form-help-text;
 
-@import 'snapcraft_p-form-validation';
+@import "snapcraft_p-form-validation";
 @include snapcraft-p-form-validation;
 
-@import 'snapcraft_p-notifications';
+@import "snapcraft_p-notifications";
 @include snapcraft-p-beta-notification;
 
-@import 'snapcraft_distro-chart';
+@import "snapcraft_distro-chart";
 @include snapcraft-distro-chart;
 
-@import 'snapcraft_p-verified';
+@import "snapcraft_p-verified";
 @include snapcraft-p-verified;
 
-@import 'utilities_crop';
+@import "utilities_crop";
 @include u-insights-crop;
 
-@import 'patterns_card';
+@import "patterns_card";
 @include insights-p-card;
 
-@import 'patterns_blog-post';
+@import "patterns_blog-post";
 @include snapcraft-p-blog-post;
 
-@import 'patterns_nested-inputs';
+@import "patterns_nested-inputs";
 @include snapcraft-p-nested-inputs;
 
-@import 'patterns_carousel';
+@import "patterns_carousel";
 @include snapcraft-p-carousel;
 
-@import 'patterns_media-object--snap';
+@import "patterns_media-object--snap";
 @include snapcraft-p-media-object--snap;
 
-@import 'snapcraft_release';
+@import "snapcraft_release";
 @include snapcraft-release;
 
-@import 'snapcraft_p-build';
+@import "snapcraft_p-build";
 @include snapcraft-p-build;
 
-@import 'patterns_case-study';
+@import "patterns_case-study";
 @include snapcraft-p-case-study;
 
-@import 'patterns_snap-row';
+@import "patterns_snap-row";
 @include snapcraft-p-snap-row;
 
-@import 'patterns_blog-list';
+@import "patterns_blog-list";
 @include snapcraft-p-blog-list;
 
-@import 'patterns_form-multiselect';
+@import "patterns_form-multiselect";
 @include snapcraft-p-multiselect;
 
-@import 'patterns_first-snap-flow';
+@import "patterns_first-snap-flow";
 @include snapcraft-p-first-snap-flow;
 
-@import 'snapcraft_p-list-step';
+@import "snapcraft_p-list-step";
 @include snapcraft-p-list-step;
 
-@import 'snapcraft_p-breadcrumbs';
+@import "snapcraft_p-breadcrumbs";
 @include snapcraft-p-breadcrumbs;
 
-@import 'patterns_code-copyable';
+@import "patterns_code-copyable";
 @include vf-p-code-copyable;
 
-@import 'snapcraft_p-code-block';
+@import "snapcraft_p-code-block";
 @include snapcraft-p-code-block;
 
-@import 'snapcraft_p-code-snippet';
+@import "snapcraft_p-code-snippet";
 @include snapcraft-p-code-snippet;
 
-@import 'snapcraft-publicise';
+@import "snapcraft-publicise";
 @include snapcraft-publicise;
+
+$baseline-color: #f00 !default;
```

Comment:
What's this stuff for? Maybe add a comment as we should avoid putting stuff in the main `styles.scss`

Reply:
Oh, that was for debugging the baseline spacing. Shouldn't be commited. Will remove it.